integrations/cdi/jpa-cdi/src/main/java/io/helidon/integrations/cdi/jpa/PersistenceUnitInfoBean.java

```diff
@@ -736,29 +736,28 @@ public final DataSource getNonJtaDataSource() {
         final Collection<String> managedClasses = persistenceUnit.getClazz();
         assert managedClasses != null;
         String name = persistenceUnit.getName();
-        if (name == null) {
-            name = "";
+        if (name == null || name.isEmpty()) {
+            name = JpaExtension.DEFAULT_PERSISTENCE_UNIT_NAME;
```

Comment:
(I missed this case on my prior (approved, merged) PR that introduced the `DEFAULT_PERSISTENCE_UNIT_NAME` field.)